src/vaadin-text-field-mixin.html

```diff
@@ -345,17 +356,20 @@
       return this.shadowRoot.querySelector('[part="value"]');
     }
 
-    /**
-     * @private
-     */
+    /** @private */
     get inputElement() {
```

Comment:
I know this was already marked as private, but as it's used quite a lot in the other files, I'd consider making this protected. Not a blocker.

Reply:
Done.

Reply:
As this is now `@protected` it should have the `@return` type too.

Reply:
Thanks, added.